feat: add issue template (#800)
eclipse-dataspaceconnector/DataSpaceConnector: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug Report
about: Create a report to help us improve.
title: ''
labels: 'bug'
assignees: ''

---

# Bug Report

## Describe the Bug

> A clear and concise description of what the bug is.

Reviewer: ```suggestion
A clear and concise description of the bug.
```

### Expected Behavior
A clear and concise description of what you expected to happen.

> ### Current Behavior

Reviewer: ```suggestion
### Observed Behavior
```

A clear and concise description of what happened instead.

## Steps To Reproduce
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

## Context Information
Add any other context about the problem here.

- Used version [e.g. EDC v1.0.0]
- OS: [e.g. iOS, Windows]
- ...

## Detailed Description
If applicable, add screenshots and logs to help explain your problem.

## Possible Implementation

> You already have an idea why this happened and how to fix it? Feel free to share your thoughts.

Reviewer: ```suggestion
You already know the root cause of the erroneous state and how to fix it? Feel free to share your thoughts.
```
